components/user/themes/UserTabs.js

```diff
@@ -46,16 +46,49 @@ export default function UserTabs({ data, BASE_URL }) {
   });
   const [tabs, setTabs] = useState(displayTabs);
   const changeTab = (e, value) => {
-    e.preventDefault();
+    e.preventDefault();    
```

Comment:
```suggestion
    e.preventDefault();
```

Reply:
does this matter, don't think so

Reply:
I see, makes sense, thanks for explaining